project/mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
+; For a complete list of options, see https://mypy.readthedocs.io/en/stable/config_file.html
+python_version = 3.9
```

Comment:
For reference:
[How do you get mypy to recognize a newer version of python?](https://stackoverflow.com/questions/60459641/how-do-you-get-mypy-to-recognize-a-newer-version-of-python)